samples/sensor/accel_polling/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		accel0 = &adxl362;
+//		accel0 = &bmi270;
```

Comment:
I was unable to get bmi270 working. Nor on nrf54l, nor nrf54h.
Maybe Kconfig is missing or driver requires sensor interrupt to be connected to a GPIO...